Automattic/jetpack
# Search: Fix Search E2E test failure for WP 5.8

#### Changes proposed in this Pull Request:
WP 5.8 introduce block widgets and the default search widget is taken over by a block search widget, the PR proposed the selectors and fixture data changes to pass search E2E test suite.

#### Jetpack product discussion
p1626890110162200-slack-jetpack-search-team

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
- Rerun [E2E test suite](https://github.com/Automattic/jetpack/actions/runs/1069659013) by clicking `re-run jobs`.
- Ensure E2E test suite passes on Github Actions

Note: 
- I have issues running it locally, it always fails before it even reaches the search suite. But it works just fine on Github Actions which needs a look afterwards.
- Also we might want to check the selectors for instant search overlay. I'll do it after this PR is merged.
